packages/terminal_size/terminal_size.0.2.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "Cryptosense <opensource@cryptosense.com>"
+authors: [
+    "Cryptosense <opensource@cryptosense.com>"
+    "Etienne Millon <etienne@cryptosense.com>"
+]
+homepage: "https://github.com/cryptosense/terminal_size"
+bug-reports: "https://github.com/cryptosense/terminal_size/issues"
+license: "BSD-2-Clause"
+dev-repo: "git+https://github.com/cryptosense/terminal_size.git"
+doc: "https://cryptosense.github.io/terminal_size/doc"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+run-test: [
+  ["dune" "runtest" "-p" name "-j" jobs]
+]
+depends: [
+  "alcotest" {with-test}
+  "dune" {build & >= "2.0.0"}
```

Comment:
Dune should not be tagged as a build dependency (see https://github.com/ocaml/dune/issues/2147)
```suggestion
  "dune" {>= "2.0.0"}
```

Reply:
Ah yes, thanks, I'll fix that upstream. This package hadn't been released in a long time.